src/Sarif.Viewer.VisualStudio/OpenLogFileCommands.cs

```diff
@@ -233,7 +233,7 @@ private void MenuItemCallback(object sender, EventArgs e)
                 logFile = openFileDialog.FileName;
             }
 
-            ErrorListService.ProcessLogFile(logFile, toolFormat);
+            ErrorListService.ProcessLogFile(logFile, SarifViewerPackage.Dte.Solution, toolFormat);
```

Comment:
>SarifViewerPackage.Dte.Solution [](start = 53, length = 31)

Here's where we first inject the solution. Unit tests of anything further down the call stack can mock the solution object (`Solution` is an interface.)